### PR TITLE
Merge feat/contributing to dev

### PR DESCRIPTION
**Pull request by puregarlic**
It is very important as an open-source organization to remove the barriers that stand before contributors. This merge requests adds the following reusable resources, which we can re-purpose for future projects:

+ An in-depth README (minus a few sections) describing getting started with the project
+ A Code of Conduct for contributors to follow
+ A Contributing file with guidelines on how to contribute
+ An Issue template to make filing bugs easier
+ A Pull Request template, so that people know what they need to include information-wise in their request.

### CODE REVIEW

File: .github/CODE-OF-CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Contributor Covenant Code of Conduct
```
Comment (astra137) [inline .github/CODE-OF-CONDUCT.md:1]: I agree to abide by this code of conduct.

File: README.md [hunk 2]
```diff
@@ -3,8 +3,122 @@
...
++ Vue
++ Vuex
++ GraphQL/Graphcool
++ Apollo
++ AT-UI
+
+### Prerequisites
+You'll need two things:
+
++ [Node Current](https://nodejs.org/en/)
```
Comment (astra137) [inline README.md:36]: Strangely named; perhaps just simplify to Node.js, or include a version number?

File: .github/CODE-OF-CONDUCT.md (hunk 1 above)
Comment (puregarlic) [reply .github/CODE-OF-CONDUCT.md:1]: Are we all good to merge?

**Review by astra137 — APPROVED** [review]
All of this seems sane to me.

File: .github/CODE-OF-CONDUCT.md (hunk 1 above)
Comment (astra137) [reply .github/CODE-OF-CONDUCT.md:1]: Yeah. GitHub did something weird though, I think I have to make a duplicate review.

**Review by astra137 — APPROVED** [review]
And now I am approving this.

**Review by astra137 — APPROVED** [review]
Okay, seriously. GitHub. This request is good.

**Comment by astra137** [comment]
My approvals are not removing the Review required notice.

**Comment by puregarlic** [comment]
@maccelerated I'll see what's up. Maybe I need to mark your comments as resolved or something.

**Comment by puregarlic** [comment]
Yeah, I'm not quite sure what's going on. I'm just going to merge the changes and hope it gets over itself

**Comment by puregarlic** [comment]
@maccelerated I solved the issue. Turns out you didn't actually have write permissions to this repository, and Github wants approval from write-enabled developers. So, I've added you to the team @multi-cell/developers, so I can add a chunk of devs to each repository and give them read/write/admin access. Now your approval is considered valid.

**Pull request merged**